projects/plugins/jetpack/extensions/blocks/sharing-buttons/edit.js

```diff
@@ -32,10 +32,14 @@ export function SharingButtonsEdit( props ) {
 				<PanelBody title={ __( 'Settings', 'jetpack' ) }>
 					<MenuItemsChoice
 						choices={ [
-							{ value: 'icon-text', label: 'Icon & Text' },
-							{ value: 'icon', label: 'Icon Only' },
-							{ value: 'text', label: 'Text Only' },
-							{ value: 'official', label: 'Official Buttons' },
+							/* translators: Sharing: Sharing button option label. */
+							{ value: 'icon-text', label: __( 'Icon & Text', 'jetpack' ) },
+							/* translators: Sharing: Sharing button option label. */
+							{ value: 'icon', label: __( 'Icon Only', 'jetpack' ) },
+							/* translators: Sharing: Sharing button option label. */
+							{ value: 'text', label: __( 'Text Only', 'jetpack' ) },
+							/* translators: Sharing: Sharing button option label. */
+							{ value: 'official', label: __( 'Official Buttons', 'jetpack' ), disabled: true },
```

Comment:
I could probably use wording like 'Official Buttons (In Development)' or something. Or leave as is.